### PR TITLE
Dict: Add complain*

### PR DESCRIPTION
**Pull request by sebweb3r**
No my flake is comlaining

_Originally posted by @sebweb3r in https://github.com/codespell-project/codespell/pull/1690#discussion_r512931323_

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -5437,6 +5437,10 @@ comittees->committees
 comitter->committer
 comitting->committing
 comittish->committish
+comlain->complain
+comlained->complained
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:5441]: Add
```suggestion
comlainer->complainer
```

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -5437,6 +5437,10 @@ comittees->committees
...
+comlaining->complaining
+comlains->complains
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:5443]:
```suggestion
comlains->complains
comlaint->complaint
comlaints->complaints
```

**Review by peternewman — CHANGES_REQUESTED** [review]
A few suggestions

**Comment by sebweb3r** [comment]
done.

**Review by peternewman — APPROVED** [review]

**Pull request merged**